### PR TITLE
ScannerCommand: Log about the local file storage at info level if in use

### PR DESCRIPTION
**Pull request by sschuberth**
Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch-si.com>

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1624)

### CODE REVIEW

File: cli/src/main/kotlin/commands/ScannerCommand.kt [hunk 1]
```diff
@@ -190,6 +182,25 @@ object ScannerCommand : CommandWithHelp() {
 
         println("Using scanner '${scanner.scannerName}' with '${ScanResultsStorage.storage.javaClass.simpleName}'.")
 
+        val localFileStorageLogFunction: ((String) -> Unit)? = when {
```
Comment (fviernau) [inline cli/src/main/kotlin/commands/ScannerCommand.kt]: I think the function is quite large already and for readability it'd be better to
extract lines 185 - 201 into a separate function. E.g.: fun  logFilesCount

Comment (sschuberth) [reply cli/src/main/kotlin/commands/ScannerCommand.kt]: I've moved out the whole scanner configuration in a separate commit.

Comment (fviernau) [reply cli/src/main/kotlin/commands/ScannerCommand.kt]: Thanks!

**Pull request merged**